server/api/googleLogin.js

```diff
@@ -71,7 +71,7 @@ async function googleSignup(req, res, username) {
  */
 async function insertGoogleUser(db, username, callback) {
   await db.run(
-    "INSERT INTO User (username) VALUES (?)",
+    "INSERT INTO User (username, is_oauth) VALUES (?, 1)",
```

Comment:
I feel like we're going to want a separate column to store the Google ID. 

WPP-137 will allow users to connect their Google Account to a username/password combo, so we need a way to store both the username and the Google ID. I'd just leave the `username` as null 

If we leave it as is, the user signs in with Google for the first time => creates a username/password combo => that username overwrites the Google ID => the user can no longer sign in with Google

Reply:
Done x2.